BitwardenShared/Core/Autofill/Services/AutofillCredentialServiceTests.swift

```diff
@@ -54,6 +54,12 @@ class AutofillCredentialServiceTests: BitwardenTestCase { // swiftlint:disable:t
             totpService: totpService,
             vaultTimeoutService: vaultTimeoutService
         )
+
+        // Wait for the `DefaultAutofillCredentialService.init` task to sync the initial set of
+        // identities for the active account, otherwise there's a potential race condition between
+        // that and the tests below.
+        waitFor { identityStore.removeAllCredentialIdentitiesCalled }
+        identityStore.removeAllCredentialIdentitiesCalled = false
```

Comment:
❓ Just a quick for myself, is there a downside to making the init for `DefaultAutofillCredentialService` async?  Or is this something an Actor could help us with?

Reply:
I think you could make it async, but I'm not sure if that would help us here. Since the init is subscribing to a publisher, we can't wait for it to complete, so how else would we know once it had finished the initial sync?

Reply:
yeah great point... I guess we could add another publisher to DefaultAutofillCredentialService which would emit when the the initial sync completes but that seems like overkill.

Reply:
Another idea would be to use a `PassthroughSubject` instead so there is no initial value but it doesn't have a buffer of the most-recent published value so maybe it's safer to leave it like it's now.

Reply:
Right, I think we want the behavior in the app to trigger immediately with the initial value though.